darwin/dataset/local_dataset.py

```diff
@@ -275,3 +281,6 @@ def __str__(self):
             f"  Number of images: {len(self.images_path)}\n"
             f"  Number of classes: {len(self.classes)}"
         )
+
+    def __add__(self, dataset):
+        return self.extend(dataset, extend_classes=True)
```

Comment:
I'm not sure I like the idea of using `__add__`, which is less explicit than `self.extend(dataset, extend_classes=True)`

Reply:
I think I used it in the auto-annotate code, but not anymore, so I can safely remove it.